src/proof/resolution_bitvector_proof.h

```diff
@@ -0,0 +1,102 @@
+/*********************                                                        */
+/*! \file resolution_bitvector_proof.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Liana Hadarean, Mathias Preiner, Guy Katz
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Bitvector proof
+ **
+ ** Bitvector proof
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__RESOLUTION__BITVECTOR__PROOF_H
+#define __CVC4__RESOLUTION__BITVECTOR__PROOF_H
+
+#include <iostream>
+#include <sstream>
+
+#include "expr/expr.h"
+#include "proof/bitvector_proof.h"
+#include "proof/theory_proof.h"
+#include "prop/bvminisat/core/Solver.h"
+
+
+namespace CVC4 {
+
+namespace prop {
+class CnfStream;
+} /* namespace CVC4::prop */
+
+namespace theory {
+namespace bv {
+class TheoryBV;
+template <class T> class TBitblaster;
+} /* namespace CVC4::theory::bv */
+} /* namespace CVC4::theory */
+
+class CnfProof;
+} /* namespace CVC4 */
+
+namespace CVC4 {
+
```

Comment:
[We should probably put this in the namespace `proof`](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#namespaces). Some existing files do not follow this style but since we are refactoring, we can introduce the proper style for the refactored parts.